src/containers/RefreshContainer.tsx

```diff
@@ -13,7 +14,10 @@ const mapStateToProps = (state: KialiAppState) => ({
 const mapDispatchToProps = (dispatch: Dispatch<any>) => {
   return {
     onSelect: bindActionCreators(UserSettingsActions.setRefreshInterval, dispatch),
-    onUpdatePollInterval: bindActionCreators(UserSettingsActions.setRefreshInterval, dispatch)
+    onUpdatePollInterval: bindActionCreators(UserSettingsActions.setRefreshInterval, dispatch),
```

Comment:
isn't it weird to have two callbacks for the same action? Shouldn't we have only one?

Reply:
Actually, I had a closer look and it seems some logic is broken. From `OverviewToolbar`, the function `this.updatePollInterval` isn't called anymore on refresh rate update, which breaks the URL update.

Reply:
Yes, I agree it should just be one callback. I will change that.

Reply:
It looks like I forgot to ask for changes in my review? Weird, I was pretty sure I did. My bad.
But the second problem I mentioned has been ignored: we now have a regression on URL handling, again.

Reply:
I expected that to have been fixed when changes were pushed.  My bad for approving and not testing it.  Sorry (again).

Reply:
@jshaughn I did resolve those: https://github.com/kiali/kiali-ui/blob/5e1f3f84e07419929ea6a19eb7e994ffb88d75bc/src/containers/RefreshContainer.tsx#L10

Reply:
@mtho11 the issue is not the duplication but rather ... "From OverviewToolbar, the function this.updatePollInterval isn't called anymore on refresh rate update, which breaks the URL update."

I'm looking into this already, I created https://issues.jboss.org/browse/KIALI-1960, if I need help I'll let you know.